cli/cmd/install.go

```diff
@@ -311,24 +311,36 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 
 	values.Stage = stage
 
-	files := []*loader.BufferedFile{
+	baseFiles := []*loader.BufferedFile{
 		{Name: chartutil.ChartfileName},
 	}
 
 	if stage == "" || stage == configStage {
 		for _, template := range templatesConfigStage {
-			files = append(files,
+			baseFiles = append(baseFiles,
 				&loader.BufferedFile{Name: template},
 			)
 		}
+		if err := charts.FilesReader(static.Templates, l5dcharts.HelmChartDirBase+"/", baseFiles); err != nil {
+			return err
+		}
+
+	}
+
+	var cpFiles []*loader.BufferedFile
+	if stage == controlPlaneStage {
```

Comment:
How about moving this as `else if` in the above if condition to make the logic simpler to understand? :thinking:

Reply:
I wanted to make the condition more explicit. But if you think it reads better, I can move it as an `else` (no additional condition is actually needed).